src/modules/px4iofirmware/px4io.h

```diff
@@ -126,6 +126,7 @@ extern uint16_t			r_page_servo_disarmed[];	/* PX4IO_PAGE_DISARMED_PWM */
 #define r_setup_sbus_rate	r_page_setup[PX4IO_P_SETUP_SBUS_RATE]
 #define r_setup_thr_fac		r_page_setup[PX4IO_P_SETUP_THR_MDL_FAC]
 #define r_setup_slew_max	r_page_setup[PX4IO_P_SETUP_MOTOR_SLEW_MAX]
+#define r_setup_slew_simplemix	r_page_setup[PX4IO_P_SETUP_SIMPLE_MIX_SLEW]
```

Comment:
Is this needed?

Reply:
No, that slipped in from a previous version where I had it controlled via a parameter as well.
Removed it and pushed.